.gitignore

```diff
@@ -1,59 +1,14 @@
-*.lcov
 *.log
-*.pid
-*.pid.lock
-*.seed
-*.tgz
 *.tsbuildinfo
-.cache
-.cache/
+.chakraIcons
+.DS_Store
 .env*
 .eslintcache
 .next
 .npm
-.pnp.*
-.tern-port
-.vscode-test
-.yarn-integrity
-.yarn/build-state.yml
-.yarn/cache
-.yarn/install-state.gz
-.yarn/unplugged
-build/Release
-coverage/
 dist/
-logs
 node_modules/
 out/
-pids
-report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
-### gitignore
-# General
-.DS_Store
-.AppleDouble
-.LSOverride
-
-# Icon must end with two \r
-Icon
-
-
-# Thumbnails
-._*
-
-# Files that might appear in the root of a volume
-.DocumentRevisions-V100
-.fseventsd
-.Spotlight-V100
-.TemporaryItems
-.Trashes
-.VolumeIcon.icns
-.com.apple.timemachine.donotpresent
-
-# Directories potentially created on remote AFP share
-.AppleDB
-.AppleDesktop
-Network Trash Folder
-Temporary Items
-.apdisk
-
 tooling/cli/dist/index.bin.js
+
+!**/.gitkeep
```

Comment:
one more, could you add this ? @grikomsn 

```
.direnv
.envrc
```

Reply:
c766156fd928aea5b871ca643d5b087e0accda68